docs/guides/response-contains-property.md

```diff
@@ -3,231 +3,337 @@ tocMaxDepth: 3
-1. Then, create a directory `plugins` inside of your root repository directory.
-  The `demo-plugin.js` file should go inside of that directory.
-1. Create a `rules` directory inside of the `plugins` directory. Place the `response-contains-property.js` file there.
-
-```js response-contains-property.js
-module.exports = ResponseContainsProperty
-
-/** @type {import('@redocly/cli').OasRule} */
-function ResponseContainsProperty (options) {
-  const mustExist = options.mustExist || [];
-  return {
-    Response: {
-      skip: (_response, key) => {
-        return !['200', '201', '202'].includes(key.toString());
-      },
-      MediaType: {
-        skip: (_media, key) => {
-          return key !== 'application/json';
-        },
-        Schema (schema, { report, location }) {
-          if (schema.type !== 'object') return;
-
-          for (let element of mustExist) {
-            if (!schema.properties[element]) {
-              report({
-                message: `Response object must have a top-level "${element}" property.`,
-                location: location.child('properties'),
-              });
-            }
-          }
-        }
-      }
-    }
-  }
-};
+```bash npm test
+npm test
+
+> acme-api@1.0.0 test
+> redocly lint
+
+validating /openapi/openapi.yaml...
+[1] openapi/components/responses/200.yaml:2:1 at #/properties
+
+Response object must contain a top-level "user_id" property.
+
+...
+
+Error was generated by the response-contains-property rule.
+
+
+[2] openapi/components/responses/201.yaml:2:1 at #/properties
+
+Response object must contain a top-level "user_id" property.
+
+...
+
+Error was generated by the response-contains-property rule.
+
+
+[3] openapi/components/responses/202.yaml:2:1 at #/properties
+
+Response object must contain a top-level "user_id" property.
+
+...
+
+Error was generated by the response-contains-property rule.
+
+
+[4] openapi/components/responses/200.yaml:2:1 at #/properties
+
+Response object must contain a top-level "user_id" property.
+
+...
+
+Error was generated by the response-contains-property rule.
+
+
+[5] openapi/components/responses/200.yaml:2:1 at #/properties
+
+Response object must contain a top-level "user_id" property.
+
+...
+
+Error was generated by the response-contains-property rule.
+
+
+/openapi/openapi.yaml: validated in 71ms
+
+❌ Validation failed with 5 errors.
+run `openapi lint --generate-ignore-file` to add all problems to the ignore file.
 ```
 
-```js demo-plugin.js
-const ResponseContainsProperty = require('./rules/response-contains-property');
-const id = 'demo';
-
-/** @type {import('@redocly/cli').CustomRulesConfig} */
-const rules = {
-  oas3: {
-    'response-contains-property': ResponseContainsProperty,
-  },
-};
-
-module.exports = {
-  id,
-  rules,
-};
+If you're using the Redocly OpenAPI extension for VS Code, this is how the lint errors look
+in the integrated terminal (Problems tab):
+
+![](./assets/response-contains-property-range.png)
```

Comment:
```suggestion
![response contains property problem](./images/response-contains-property-range.png)
```
issue: Link was broken, and always include an alt-text description for accessibility.